README.md

```diff
@@ -30,6 +30,7 @@
 ## Readme Generator
 
 - [Get Readme](https://github.com/luctst/get-readme#readme) - CLI tool to generate an esthetic readme file 🔖.
+- [Plantek README Manager](https://gitlab.com/MarleyPlant/readme-manager)
```

Comment:
Please add a short description of the tool in the below format:
Tool Name - Description

Reply:
Also according to alphabetical order, this should come after **GitHub Profilinator**